.circleci/config.yml

```diff
@@ -434,12 +465,15 @@ jobs:
   deploy-maven-jni-snapshot-mac-arm64:
     executor: mac-arm64
     steps:
-      - deploy-maven-jni-snapshot-mac
+      - deploy-maven-jni-snapshot-mac:
+          arch: arm64
 
   deploy-maven-jni-snapshot-mac-x86_64:
-    executor: mac-x86_64
+    executor: mac-arm64
     steps:
-      - deploy-maven-jni-snapshot-mac
+      - macos/install-rosetta
+      - deploy-maven-jni-snapshot-mac:
+          arch: amd64
```

Comment:
This is now horribly confusing to read but nothing we can do I guess!

Can we roll the `mac-x86_64` jobs into the same jobs as for `mac-arm64` and just call it `<>-mac-all`? This would save us a large set of jobs.

Reply:
We can rename the Mac executor to be `mac-all`.
I doubt we gain anything from trying to combine the jobs since they have to be built for their architecture all the way down.